README.md

```diff
@@ -1 +1,162 @@
-# federation-support
\ No newline at end of file
+# federation-support
+
+A module to use the federation feature in `mercurius`.
+
+This module extract the federation parts from `mercurius` to be used as external plugin.
```

Comment:
I think the end goal is to split the implementation of Apollo Federation out from Mercurius.